### ISSUE TITLE
Upgrade to Storybook v7

### ISSUE THREAD

**Issue by thibaudcolas** [torchbox/storybook-django#10]
Currently in the "late alpha" phase according to https://storybook.js.org/blog/first-class-vite-support-in-storybook/.

**Comment by lb-** [comment]
We are using this package with Storybook 7 in Wagtail, not sure if this issue is fixed.

**Comment by thibaudcolas** [comment]
Yep, the package is already on v7, and support was of the "just works" kind as far as package users!
